python/paddle/fluid/tests/unittests/dygraph_to_static/CMakeLists.txt

```diff
@@ -10,6 +10,10 @@ if(NOT ON_INFER)
     set_tests_properties(test_lac PROPERTIES TIMEOUT 120)
 endif()
 
+if(WIN32 AND NOT WITH_GPU)
+    list(REMOVE_ITEM TEST_OPS test_resnet_amp) # disable on Windows CPU CI for timeout
```

Comment:
Why Windows cpu timeout but not Linux/Mac cpu

Reply:
Firstly, AMP only supports CUDA, so we don't need to run this unittest on CPU CI. Because it will downgrade into FP32 mode.

Secondly, I can't reproduce this problem on Windows Debug machine 1. That's wired.